controlpanel/frontend/jinja2/reset.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+
+{% set page_title = "Hard Reset Home Directory" %}
+{% set page_name = "reset" %}
+
+{% block content %}
+  <div class="markdown-container" id="reset-container">
+    <h1>Home Directory Reset</h1>
+    <p>Even the best curated home directories can get into a bad
+    state (for example, a broken conda environment or Jupyter and RStudio won't
+    start).</p>
+
+    <p>Reset your home directory to recreate your conda environment and shell
+    settings to a clean state. Technical details of this process can be
+    <a href="">found here</a>.</p>
```

Comment:
should this anchor have an empty href? I've just started to have a look at this PR so apologies if it's set dynamically somehow 🤷‍♂️

Reply:
Nope... it's not yet written. ;-)

Reply:
Actually, I know where I could point it.

Reply:
Resolved in latest commit.